.github/workflows/pre-commit.yml

```diff
@@ -13,3 +13,5 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v3
       - uses: pre-commit/action@v2.0.3
+      - run: pip install flake8 flake8-variables-names
+      - run: flake8 --select=VNE,N802 --show-source --statistics --ignore VNE001,VNE003 .
```

Comment:
Not that I'm against it, but what's the reason for this addition?

Reply:
I wanted to enforce method and variable names, but it doesn't seem to be working as intended (should fail on the old functions) and other headaches. Might take it out tomorrow then.